components/x-follow-button/package.json

```diff
@@ -26,5 +27,12 @@
   "dependencies": {
     "@financial-times/x-engine": "file:../../packages/x-engine",
     "classnames": "^2.2.6"
+  },
+  "engines": {
+    "node": "12.x",
+    "npm": "7.x || 8.x"
```

Comment:
**question:** Since we should be upgrading to npm 8 to get ready to migrate to bower, would it make sense to remove the 7.x for each engines object?